test/espnet2/enh/test_espnet_model.py

```diff
@@ -269,7 +234,5 @@ def test_forward_with_beamformer_net(
         "speech_mix": inputs,
         "speech_mix_lengths": ilens,
         **{"speech_ref{}".format(i + 1): speech_refs[i] for i in range(num_spk)},
-        "noise_ref1": noise_ref1,
-        "dereverb_ref1": dereverb_ref1,
```

Comment:
Better to preserve these lines for later support of denoising and dereverberation losses.

Reply:
Done.